packages/css/src/plugins/atomicify-rules.tsx

```diff
@@ -156,9 +164,20 @@ const atomicifyRule = (node: Rule, opts: AtomicifyOpts): Rule[] => {
  * @param node
  * @param opts
  */
-const atomicifyAtRule = (node: AtRule, opts: AtomicifyOpts): AtRule => {
-  const children: Node[] = [];
-  const newNode = node.clone({ nodes: children });
+const atomicifyAtRule = (node: AtRule, opts: AtomicifyOpts, postcss: Postcss): AtRule => {
+  const children: ChildNode[] = [];
+  const newNode = postcss.atRule({
```

Comment:
How come the original `node.clone({ nodes: children })` isn't used here? It should be part of the `AtRule` API

Reply:
I was initially having issues after the update, coming back to this now the use of clone seems to work as it should, have now updated as such